lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1076,5 +1076,12 @@ OpFoldResult PrimDtypeOp::fold(ArrayRef<Attribute> operands) {
   }
   return nullptr;
 }
+
+OpFoldResult AtenIntTensorOp::fold(ArrayRef<Attribute> operands) {
```

Comment:
Comment here: If an int-elemental value is converted to a zero-d tensor and passed on to `aten.Int.Tensor` op, it's better to remove the `aten.Int.Tensor` op and yield the int-elemental value.